indigo_api/migrations/0130_migrate_namespaces.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.12 on 2020-05-12 15:26
+
+import json
+from reversion import revisions as reversion
+from reversion.models import Version
+
+from django.contrib.auth.models import User
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+
+from indigo_api.data_migrations import UpdateAKNNamespace
+
+
+def forward(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    Document = apps.get_model("indigo_api", "Document")
+    ct_doc = ContentType.objects.get_for_model(Document)
+    user = User.objects.filter(pk=1).first()
+    migration = UpdateAKNNamespace()
+
+    for document in Document.objects.using(db_alias).all():
+        update = False
+        xml = migration.update_namespace(document.document_xml)
+        if xml != document.document_xml:
+            document.reset_xml(xml, from_model=True)
+            update = True
+
+        if update:
```

Comment:
This implies we can roll this change back. We can't. So we don't need any of this.